File: Tareas/Tarea8/Tarea8.java
/*  
Nombre: Axel Ramon Ortiz Flores Matricula: 1819000 Hora: V6
Tarea 8:  Conversión de Temperatura 
*/

import javax.swing.JOptionPane;
public class Tarea8{
	public static void main(String[] args){
		String entrada;
		String[] options ={"Ninguna","C >> F","F >> C"};
		float x;
		double f, c;
		
		int v = JOptionPane.showOptionDialog(null, "Ingrese la opcion deseada: ", "Conversion de Temperatura", 
		JOptionPane.YES_NO_CANCEL_OPTION, 
		JOptionPane.QUESTION_MESSAGE, 
		null,
		options,
		options[0]);

		if(v==0){
			JOptionPane.showMessageDialog(null, "No ha elegido ninguna opcion de conversion, Gracias!!");
		}else if(v==1){
				entrada= JOptionPane.showInputDialog("Ingrese los grados Celcius a convertir:");
			    x= Float.parseFloat(entrada);
			    f= ((x*1.8)+32);
			    JOptionPane.showMessageDialog(null, x + " grados Celcius es igual a " + String.format("%.2f", f)+" en grados Farenheit");
			}else if(v==2){
				entrada= JOptionPane.showInputDialog("Ingrese los grados Celcius a convertir:");
				x= Float.parseFloat(entrada);
				c=(x-32)/1.8;
				JOptionPane.showMessageDialog(null, x + "grados Farenheit es igual a "+ String.format("%.2f", c)+" en grados Celcius");
			}
		}
	}

File: Tareas/Tarea8/Tarea8.rb
=begin 
Nombre: Axel Ramon Ortiz Flores  Matr.1819000  Hora:V6
Tarea 8: Conversión de Temperatura
=end

print("\t\t .: Menu :.\n\n\t\t1. °C -> °F\n\t\t2. °F -> °C\n")
print("\nEn base al menu anterior ingrese el numero de la conversion que desea: ")
n= gets().to_i

	if(n==1) then
		print("\nIngrese la temperatura en grados Centigrados a convertir: ")
        x= gets().to_f
        F=(x* 1.8)+ 32
        puts("\nLa temperatura de #{x} °C es igual a #{F}°F")
	else
		if(n==2) then
			print("\nIngrese la temperatura en  grados Farenheit a convertir: ")
        	x= gets().to_f 
        	C=(x-32)*1.8 
            puts("\nLa temperatura de #{x} °F es igual a #{C} °C")
        else
        	puts("\nDebe ingresar un numero del menu dado anteriormente")
        end
	end